### PR TITLE
PS-3740 LP #1716507: innodb_flush_method needs updating in documentation

### PR DESCRIPTION
**Pull request by patrickbirch**
modified files:
scalability/innodb_io
changed the default value
changed the allowed values to match 5.6

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval

**Review by nastena1606 — APPROVED** [review]

**Pull request merged**
